smartcosmos-framework-monitoring/src/main/java/net/smartcosmos/metrics/MetricsProperties.java

```diff
@@ -0,0 +1,43 @@
+package net.smartcosmos.metrics;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Getter;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import static net.smartcosmos.metrics.MetricsConfiguration.PREFIX_ENDPOINTS_METRICS;
+
+/**
+ * Properties for the default Metrics endpoint configuration.
+ */
+@ConfigurationProperties(prefix = PREFIX_ENDPOINTS_METRICS)
+@Data
+public class MetricsProperties {
+
+    /**
+     * Mark if the endpoint exposes sensitive information.
+     */
+    private Boolean sensitive = true;
+
+    /**
+     * Endpoint URL path.
+     */
+    @NotNull
+    @Pattern(regexp = "/\\w+", message = "Path must start with '/' and must contain only letters, numbers and '_'")
+    @Getter(AccessLevel.NONE)
+    private String path = "/default_metrics";
+
+    /**
+     * Enable the default metrics endpoint.
+     */
+    private Boolean enabled = false;
```

Comment:
Spring does standardize on the `enabled` keyword, and the variable needs to match.